python/remotivelabs-broker/docker-build.sh

```diff
@@ -1,21 +1,21 @@
 #!/bin/bash
-
+#
+# Generates code from proto files, builds package and generates documentation
+#
 set -e
 
-# Generates code from proto files, builds package and generates documentation
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
We obviously use bash in the shebang, but this one is better for sh compatibility 
> SCRIPT_DIR=$(cd -- "$(dirname -- "$0")" && pwd)

Reply:
fine by me